release notes/v1.1.0.md

```diff
@@ -0,0 +1,115 @@
+
+## Bug fixes
+
+- [#4794](https://github.com/grafana/k6/pull/4794) Fixes race conditions from stringifying types in `k6/browser`.
+- [#4809](https://github.com/grafana/k6/pull/4809) Fixes the `locator.fill` method when used on react based websites.
+- [#4831](https://github.com/grafana/k6/pull/4831) Fixes the Dockerfile for k8s use by setting the user to `12345` instead of `k6` to avoid having to work with `runAsUser` in the pod manifest file.
+- [#4845](https://github.com/grafana/k6/pull/4845) Fixes an infrequent panic when `click` is called.
+
+## Maintenance and internal improvements
+
+- [#4608](https://github.com/grafana/k6/pull/4608) Enables the 'copyloopvar' linter.
+- [#4744](https://github.com/grafana/k6/pull/4744) Updates the 'golangci-lint' linter to v2.
+- [#4746](https://github.com/grafana/k6/pull/4746) Adds a collection of small `k6/browser` performance improvements.
+- [#4750](https://github.com/grafana/k6/pull/4750) Fixes the lint GHA.
+- [#4775](https://github.com/grafana/k6/pull/4775) Updates the version of the golangci-lint GHA.
+- [#4784](https://github.com/grafana/k6/pull/4784) Fixes the golangci-lint version detection and execution. Thanks, @tbourrely, for the contribution!
+- [#4785](https://github.com/grafana/k6/pull/4785) Updates the `chromedp/cdproto` dependency and adjusts the Browser module accordingly.
+- [#4800](https://github.com/grafana/k6/pull/4800) Allows un/marshaling of invalid UTF-8 when using JSONv2 within the Browser module. 
+- [#4802](https://github.com/grafana/k6/pull/4802) Fixes the `examples/grpc_server` and updates its dependencies. 
+- [#4822](https://github.com/grafana/k6/pull/4822) Uses the `CODECOV_TOKEN` variable for GH Workflows from Vault.
+- [#4831](https://github.com/grafana/k6/pull/4831) Fixes the default user defined in the Dockerfile.
+- [#4833](https://github.com/grafana/k6/pull/4833) Retrieves secrets from Vault and adjusts the 'k6packager' deployment process.
+- [#4837](https://github.com/grafana/k6/pull/4837), [#4847](https://github.com/grafana/k6/pull/4847) Prevent Codecov from running on forks.
+- [#4848](https://github.com/grafana/k6/pull/4848) Enables CI pipelines to be executed also on ARM (`ubuntu-24.04-arm`). Thanks, @nadiamoe, for the contribution!
+- [#4855](https://github.com/grafana/k6/pull/4855) Adds some changes to make Browser tests more stable.
+- [#4780](https://github.com/grafana/k6/pull/4780), [#4781](https://github.com/grafana/k6/pull/4781), [#4782](https://github.com/grafana/k6/pull/4782), [#4786](https://github.com/grafana/k6/pull/4786), [#4798](https://github.com/grafana/k6/pull/4798), [#4816](https://github.com/grafana/k6/pull/4816), [#4821](https://github.com/grafana/k6/pull/4821), [#4835](https://github.com/grafana/k6/pull/4835), [#4840](https://github.com/grafana/k6/pull/4840), [#4859](https://github.com/grafana/k6/pull/4859) Update direct dependencies.
```

Comment:
```suggestion
- [#4780](https://github.com/grafana/k6/pull/4780), [#4781](https://github.com/grafana/k6/pull/4781), [#4782](https://github.com/grafana/k6/pull/4782), [#4786](https://github.com/grafana/k6/pull/4786), [#4798](https://github.com/grafana/k6/pull/4798), [#4816](https://github.com/grafana/k6/pull/4816), [#4821](https://github.com/grafana/k6/pull/4821), [#4835](https://github.com/grafana/k6/pull/4835), [#4840](https://github.com/grafana/k6/pull/4840) Update direct dependencies.
```

This is the Sobek/goja update, but it was reverted. I do not think we need to explain this in teh release notes though